harness/determined/cli/trial.py

```diff
@@ -9,11 +9,12 @@
 from termcolor import colored
 
 from determined import cli
-from determined.cli import render
+from determined.cli import errors, render
 from determined.cli.master import format_log_entry
 from determined.common import api
 from determined.common.api import authentication, bindings
 from determined.common.declarative_argparse import Arg, ArgsDescription, Cmd, Group, string_to_bool
+from determined.common.experimental.checkpoint import CheckpointState
```

Comment:
The SDK itself gets a pass, but everywhere else we should reference symbols from outside the `determined.common.experimental` (I think it already throws a warning if you import like this). `CheckpointState` is aliased into `client`.

To be forward-looking ([here's our doc](https://hpe.sharepoint.com/:w:/r/teams/detai/_layouts/15/Doc.aspx?sourcedoc=%7BCCA17AD5-E1DA-4660-AE14-97F22A58C1CB%7D&file=python%20code%20style.docx&action=default&mobileredirect=true), https://google.github.io/styleguide/pyguide.html#22-imports the Google style guide), please only import package / module names.